src/main/java/org/sonarqube/A.java

```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
+    message.length();
+
     return message;
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: The return value of "length" must be used. ([AWBEzrmbI9pSNYZJmpeG](foo))
([See it in SonarQube](bar)

[ISSUE_KEY]: AWBEzrmbI9pSNYZJmpeG